desktop/apps/auction_support/client/bid_form.coffee

```diff
@@ -12,10 +12,12 @@ module.exports = class BidForm extends ErrorHandlingForm
 
   timesPolledForBidPlacement: 0
   maxTimesPolledForBidPlacement: sd.MAX_POLLS_FOR_MAX_BIDS
-  errors:
+  errors: -> {
```

Comment:
cc @craigspaeth - I'm wondering if this was the right approach here. The nesting functions seems like a smell, but it works and has tests. Otherwise I think this PR is good to go.